source/plugins/freeze.plugin.ts

```diff
@@ -52,9 +58,10 @@ class FreezeTaw extends Plugin.ChatCommand
 		}
 
 		let memberRole = this.plugin.getMemberRole( message.guild );
-		let modRole = this.plugin.getModRole( message.guild );
+		let chatModRole = this.plugin.getChatModRole( message.guild );
```

Comment:
immutable variables should use `const`
also please fix inconsistent spacing.
```suggestion
		const chatModRole = this.plugin.getChatModRole(message.guild);
```